providers/install.rb

```diff
@@ -52,43 +54,63 @@ def download
 end
 
 def unpack
-  install_dir = "#{new_resource.base_name}#{new_resource.version}"
+  install_dir = "#{new_resource.base_name}#{new_resource.version}" # rubocop:disable Lint/UselessAssignment
   case new_resource.artifact_type
-    when "tar.gz",".tgz"
-      execute %(cd #{new_resource.download_dir} ; mkdir -p '#{install_dir}' ; tar zxf '#{@tarball}' --strip-components=1 -C '#{install_dir}' --no-same-owner)
-    else
-      raise Chef::Exceptions::UnsupportedAction, "Current package type #{new_resource.artifact_type} is unsupported"
+  when 'tar.gz', '.tgz'
+    execute %w(
```

Comment:
It seems that `%w()` is for non-interpolated strings, and so many of these commands are now broken:
```
           ================================================================================
           Error executing action `run` on resource 'redisio_install[redis-installation]'
           ================================================================================

           RuntimeError
           ------------
           cd #{new_resource.download_dir} ; mkdir -p '#{install_dir}' ; tar zxf '#{@tarball}' --strip-components=1 -C '#{install_dir}' --no-same-owner
```